src/common/entities/otp.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, PrimaryKey, Property, EntityRepositoryType } from "@mikro-orm/core";
+
+import { OtpRepository } from "@/otp/otp.repository";
+
+import { CustomBaseEntity } from "./custom-base.entity";
+
+@Entity({ tableName: "otps", repository: () => OtpRepository })
```

Comment:
Is this for the reset password functionality?
Can you please name it something more relevant? Otps is very vague